engine/src/main/java/org/terasology/particles/ParticleSystemManager.java

```diff
@@ -24,17 +24,35 @@
 import java.util.stream.Stream;
 
 /**
- * Component system responsible for keeping track of all ParticleSystem components and updating them.
+ * Component system responsible for keeping track of all {@link org.terasology.particles.components.ParticleEmitterComponent} components and updating them.
  * Also maintains a registry of generator and affector functions to be used when processing generators
  * and affectors during a particle system update.
  */
 
 @API
 public interface ParticleSystemManager {
 
+    /**
+     * Registers an Affector function to be used when updating particle systems that have the registered affector as a component.
+     *
+     * @param affectorFunction The affector function to be used.
+     */
     void registerAffectorFunction(AffectorFunction affectorFunction);
 
+    /**
+     * Registers an Affector function to be used when updating particle systems that have the registered affector as a component.
```

Comment:
Registers an -affector- function? Either I'm missing something, or you did not update the description after copy-pasting it.